stm32loader.cpp

```diff
@@ -1,161 +1,173 @@
 #include "stm32loader.h"
 
-char getACKByte() {
-	return STM32_COMM_ACK;
+char getACKByte()
+{
+    return STM32_COMM_ACK;
```

Comment:
in diesem Fall koenntest du hier sogar constexpr funktionen verwenden. Dadurch kann der Compiler die Funktion so optimieren, dass alles zu Compile-Zeit aufgeloeßt wird. 
``` constexpr char getACKByte(){
return ACK;
}
```

Jedoch frag ich mich, wo diese Funktion dazu gehoert. Ist ja in keiner Klasse drin